exp/term/ansi/wrap.go

```diff
@@ -0,0 +1,246 @@
+package ansi
+
+import (
+	"bytes"
+	"unicode"
+	"unicode/utf8"
+
+	. "github.com/charmbracelet/x/exp/term/ansi/parser"
```

Comment:
Unused, so should this be removed?

Reply:
It's dot-importing the `parser` package which contains the state machine. Renamed the import and removed the dot-import.